.github/workflows/export_course_lesson_data_report.yml

```diff
@@ -0,0 +1,59 @@
+name: Export course lesson data report to BigQuery
+
+on: workflow_dispatch
+
+jobs:
+  deploy:
+    runs-on: ubuntu-20.04
+    container: google/cloud-sdk:latest
+    steps:
+      - name: Auth with gcloud
+        uses: google-github-actions/setup-gcloud@master
+        with:
+          service_account_key: ${{ secrets.GOOGLE_SERVICE_ACCOUNT_BASE64 }}
+
+      - name: Install CloudFoundry CLI
+        shell: bash
+        run: |
+          apt-get install -y wget postgresql-client
+          wget -q -O - https://packages.cloudfoundry.org/debian/cli.cloudfoundry.org.key | apt-key add -
+          echo "deb https://packages.cloudfoundry.org/debian stable main" | tee /etc/apt/sources.list.d/cloudfoundry-cli.list
+          apt-get update
+          apt-get install cf7-cli
+      - name: Install conduit plugin
+        shell: bash
+        run: |
+          cf install-plugin conduit -f
+
+      - name: Generate dashboard report
+        shell: bash
+        env:
+          PAAS_ORGANISATION: dfe
+          PAAS_SPACE: early-careers-framework-staging
+          ENV_STUB: staging
```

Comment:
You can see how it works in a review app too, right?

Reply:
Or maybe not, cause a workflow might need to be on develop to run it :P 
How about using `dev` then?

Reply:
What is the difference in doing it with dev over staging if we have to merge the PR to test it?

Reply:
Dev gets deployed automatically, staging you need to deploy yourself :D

Reply:
updated